src/main/java/org/apache/datasketches/GenericSortedView.java

```diff
@@ -31,44 +31,64 @@
    * Gets the quantile based on the given normalized rank, and the given search criterion.
    * @param normalizedRank the given normalized rank, which must be in the range [0.0, 1.0].
    * @param searchCrit the given search criterion to use.
-   * @return the associated quantile value.
+   * @return the associated quantile item.
    */
   T getQuantile(double normalizedRank, QuantileSearchCriteria searchCrit);
 
   /**
-   * Gets the normalized rank based on the given quantile value.
-   * @param value the given quantile value
+   * Gets the normalized rank based on the given quantile item.
+   * @param item the given quantile item
    * @param searchCrit the given search criterion to use.
    * @return the normalized rank, which is a number in the range [0.0, 1.0].
    */
-  double getRank(T value, QuantileSearchCriteria searchCrit);
+  double getRank(T item, QuantileSearchCriteria searchCrit);
 
   /**
    * Returns an array of values where each value is a number in the range [0.0, 1.0].
    * The size of this array is one larger than the size of the input splitPoints array.
    *
-   * <p>If <i>isCdf</i> is true, the points in the returned array are monotonically increasing and end with the
+   * <p>The points in the returned array are monotonically increasing and end with the
    * value 1.0. Each value represents a point along the cumulative distribution function that approximates
    * the CDF of the input data stream. Therefore, each point represents the fractional density of the distribution
-   * between from zero. For example, if one of the returned values is 0.5, then the splitPoint corresponding to that
-   * value would be the median of the distribution.</p>
+   * from zero to the given point. For example, if one of the returned values is 0.5, then the splitPoint corresponding
+   * to that value would be the median of the distribution.</p>
    *
-   * <p>If <i>isCdf</i> is false, the points in the returned array are not monotonic and represent the discrete
-   * derivative of the CDF, or the Probability Mass Function (PMF). Each returned point represents the fractional
+   * @param splitPoints the given array of quantile values or splitPoints. This is a sorted, monotonic array of unique
+   * values in the range of (minValue, maxValue). This array should not include either the minValue or the maxValue.
```

Comment:
still value here

Reply:
Not sure what "still value here" means but I already corrected the "should not include" as stated above.

Reply:
I mean the mention of minValue, maxValue

Reply:
Oh!  You mean the use of the word "value".  I'm in the process of fixing.